### PR TITLE
build: do not require go-test-teamcity in the user's path

### PR DESCRIPTION
**Pull request by petermattis**

### CODE REVIEW

File: build/teamcity-nightly-acceptance.sh [hunk 1]
```diff
@@ -119,4 +119,4 @@ build/builder.sh make TYPE=$TYPE testbuild PKG=$PKG
 cd $PKG
 # shellcheck disable=SC2086
 ./${PKG##*/}.test -l "$TMPDIR" -test.v -test.timeout $TESTTIMEOUT -show-logs -remote -key-name terraform \
-  $COCKROACH_EXTRA_FLAGS | go-test-teamcity
+  $COCKROACH_EXTRA_FLAGS | $(command -v go-test-teamcity 2>/dev/null || echo cat)
```
Comment (benesch) [inline build/teamcity-nightly-acceptance.sh]: I think shellcheck would want the command substitution quoted `"$(command -v...)"`.

**Review by benesch — APPROVED** [review]
Can you remove item three from the instructions at the top of the file?

File: build/teamcity-nightly-acceptance.sh (hunk 1 above)
Comment (petermattis) [reply build/teamcity-nightly-acceptance.sh]: Done.

**Comment by benesch** [comment]
:shipit:

**Pull request merged**